src/main/java/stringcalculator/domain/Calculator.java

```diff
@@ -0,0 +1,12 @@
+package stringcalculator.domain;
+
+import java.util.List;
+
+public class Calculator {
+
+    public int calculate(List<String> number) {
+        int sum = number.stream().mapToInt(Integer::parseInt).sum();
+        return sum;
```

Comment:
아래와 같이 굳이 변수를 선언할 필요 없는 것 같아요. 🤔
또한 되도록 stream 함수를 사용할 때는 한 줄당 하나의 메서드만 호출하도록 줄 바꿈을 적절하게 해주세요! 😃 
이렇게 한 줄에 하나의 메서드만 호출하도록 변경함으로써 IntelliJ는 아래의 사진과 같이 현재 `type`을 명시해주게 됩니다. 😃

```suggestion
        return number.stream()
                .mapToInt(Integer::parseInt)
                .sum();
```

![image](https://user-images.githubusercontent.com/47850258/152642188-f265bfc9-daf6-43d3-8895-8e8ced863887.png)